testdata/project-v4/PROJECT

```diff
@@ -4,7 +4,7 @@
 # More info: https://book.kubebuilder.io/reference/project-config.html
 domain: testproject.org
 layout:
-- go.kubebuilder.io/v4
+- go.kubebuilder.io/v3
```

Comment:
We cannot change it here.
Need be via the GitHub action otherwise it will fail in the testdata checks

Reply:
Yep I can see

> Run cd testdata/project-v4 && kubebuilder alpha generate
level=warning msg="Detected 'go.kubebuilder.io/v3' in PROJECT file."
level=warning msg="Kubebuilder v[4](https://github.com/kubernetes-sigs/kubebuilder/actions/runs/13903415389/job/38953378793?pr=4572#step:7:5) no longer supports this. It will be replaced with 'go.kubebuilder.io/v4'."
level=info msg="PROJECT file updated successfully."